libraries/AP_HAL/system.h

```diff
@@ -18,6 +18,12 @@ uint16_t millis16();
 uint64_t micros64();
 uint64_t millis64();
 
+uint32_t native_micros();
```

Comment:
again, why the "native" calls?
ahh, is this for SITL, to get at wall clock time?

Reply:
yes